docs/reference/feel/language-guide/feel-unary-tests.md

```diff
@@ -3,56 +3,105 @@ id: feel-unary-tests
-* The expression evaluates to a value and the input value is equal to that value.
+* the expression evaluates to `true` when the input value is applied to it
+* the expression evaluates to a list, and the input value is equal to at least one of the values in
+  that list
+* the expression evaluates to a value, and the input value is equal to that value
+* the expression is equal to `-` (a dash)
 
 ### Comparison
 
-Compare the input value to `x`.
+Compares the input value with a given value. Both values must be of the same type.
+
+The input value is passed implicitly as the first argument of the operator.
+
+<table>
+  <tr>
+    <th>Operator</th>
+    <th>Description</th>
+    <th>Supported types</th>
+  </tr>
+
+  <tr>
+    <td>(none)</td>
+    <td>equal to</td>
+    <td>any</td>
+  </tr>
+
+  <tr>
+    <td>&lt;</td>
+    <td>less than</td>
+    <td>number, date, time, date-time, duration</td>
+  </tr>
+
+  <tr>
+    <td>&lt;=</td>
+    <td>less than or equal to</td>
+    <td>number, date, time, date-time, duration</td>
+  </tr>
+
+  <tr>
+    <td>&gt;</td>
+    <td>greater than</td>
+    <td>number, date, time, date-time, duration</td>
+  </tr>
+
+  <tr>
+    <td>&gt;=</td>
+    <td>greater than or equal</td>
+    <td>number, date, time, date-time, duration</td>
+  </tr>
+
+</table>
 
-| operator | symbol | example |
-|----------|-----------------|---------|
-| equal to | (none) | `"valid"` |
-| less than | `<`  | `< 10` |
-| less than or equal | `<=` | `<= 10` |
-| greater than | `>` | `> 10` |
-| greater than or equal | `>=` | `>= 10` |
+```js
+"valid"
+
+< 10
+
+<= date("2020-04-06")
 
-* Less than/greater than are only supported for:
-  * Number
-  * Date
-  * Time
-  * Date-time
-  * Year-month-duration
-  * Day-time-duration
+> time("08:00:00")
+
+>= duration("P5D")
+```
 
 ### Interval
 
-Test if the input value is within the interval `x` and `y`.
+Checks if the input value is within a given interval between `x` and `y`.
+
+An interval have two boundaries that can be open `(x..y)` / `]x..y[` or closed `[x..y]`. If a
+boundary is closed then it includes the given value (i.e. less/greater than or equal). Otherwise, it
+excludes the value (i.e. less/greater than).
 
-An interval can be open `(x..y)` / `]x..y[` or closed `[x..y]`. If the interval is open, the value is not included.
+The input value is passed implicitly to the operator.
 
 ```js
 (2..5)
 // input > 2 and input < 5
 
+]2..5[
+// input > 2 and input < 5
+
 [2..5]
 // input >= 2 and input <= 5
 
 (2..5]
 // input > 2 and input <= 5
 ```
 
-### Disjunction
+### Disjunction / Or
 
-Test if at least of the expressions is `true`.
+Combines multiple unary-test expressions following the ternary logic.
 
+* returns `true` if at least one unary-test evaluates to `true`
+* otherwise, it returns `false`
+  http://localhost:3000/docs/reference/feel/language-guide/feel-functions/#invocation
```

Comment:
Is this meant to be linked to text?

Reply:
Nope. The links should not be there :sweat_smile: